curvefs/src/client/fuse_client.cpp

```diff
@@ -31,6 +31,7 @@
 #include <set>
 #include <unordered_map>
 #include <utility>
+#include <boost/algorithm/string.hpp>
```

Comment:
Why introduce boost？ It is inappropriate to introduce these functions of string

Reply:
has removed boost.

Reply:
has removed.